build.gradle

```diff
@@ -397,21 +398,48 @@ task mapIntermediaryJar(dependsOn: [downloadMcLibs, build]) {
 	}
 }
 
-task mapNamedJar(dependsOn: mapIntermediaryJar) {
+task mapYarnJar(dependsOn: mapIntermediaryJar) {
 	inputs.files downloadMcLibs.outputs.files.files
-	outputs.file(namedJar)
+	outputs.file(yarnJar)
 
 	//Force the task to always run
 	outputs.upToDateWhen {false}
 
 	doLast {
-		logger.lifecycle(":mapping minecraft to named")
+		logger.lifecycle(":mapping minecraft to yarn")
 
 		def tinyInput = new File("build/libs/yarn-tiny-${yarnVersion}.gz")
-		mapJar(namedJar, intermediaryJar, tinyInput, libraries, "intermediary", "named")
+		mapJar(yarnJar, intermediaryJar, tinyInput, libraries, "intermediary", "named")
+	}
+}
+
+task mapNamedJar(dependsOn: [mergeTiny, mapIntermediaryJar]) {
+	inputs.files downloadMcLibs.outputs.files.files
+	outputs.file(namedJar)
+
+	//Force the task to always run
+	outputs.upToDateWhen { false }
+
+	doLast {
+		logger.lifecycle(":mapping minecraft to named")
+
+		def resultMappings = mergeTiny.fileOutput
+		def namedWithEnum = new File(tempDir, "named-with-enum.tiny")
+
+		logger.lifecycle(":seeking auto-mappable fields")
```

Comment:
I would make a separate task to generate this mapping file.

Reply:
Didn't you reject the generation for its potential risk to infringe on mojang's copyright?

Reply:
No need to ship it, just make it a task that generates it, its not the end of the world, I just think it makes more sense.

Reply:
Moved to another task.